internal/app/connectconformance/process.go

```diff
@@ -70,13 +70,34 @@ func runCommand(command []string) processStarter {
 		if err := cmd.Start(); err != nil {
 			return nil, err
 		}
-		return (*cmdProcess)(cmd), nil
+		cmdProc := &cmdProcess{
+			cmd:  cmd,
+			done: make(chan struct{}),
+		}
+		go func() {
+			// cmd.Wait can only be called once. So we call it from this goroutine
+			// and then publish the result so it can be read via cmdProc.result()
```

Comment:
This was a fun one. I find it rather strange that `Wait` should only be called once. There's no other way to retrieve the status of the program. So if you have other goroutines that want to watch for when the program terminates, you have to do something like this 🤷